src/sql/workbench/browser/modelComponents/loadingComponent.component.ts

```diff
@@ -17,11 +17,13 @@ import * as nls from 'vs/nls';
 @Component({
 	selector: 'modelview-loadingComponent',
 	template: `
-		<div class="modelview-loadingComponent-container" role="alert" aria-busy="true" *ngIf="loading">
-			<div class="modelview-loadingComponent-spinner" *ngIf="loading" [title]=_loadingTitle #spinnerElement></div>
+		<div [style.display]="display">
```

Comment:
Does this change impact other components? Have we tested them & verified this doesn't cause unexpected behavior? I think we need to be careful when changing such a common component, please ask the rest of the team if this could impact them.

Reply:
I went through and verified that other places don't appear to be impacted. Wrapping something in a div like this generally shouldn't impact anything.

This is necessary because I needed to show/hide both the loading image itself and the child component and I didn't want to add the display property to both of them when wrapping in a div seems cleaner.

Reply:
Alright, ended up just wrapping the components in another div in the dashboard doing the change in the modelview code was riskier. At some point I imagine we'll want to go in and make it so this uses the display property though.